### DISCUSSION TITLE
This is internal CMI command.... How to disable?

### DISCUSSION THREAD

**Discussion by iBuseWinner** [Zrips/CMI#5559] [Q&A] [answered]
When I used / command (just write / in chat) I get this message and below "help" message. How to disable "This is internal CMI command"?????
![image](https://user-images.githubusercontent.com/44753872/117629088-7911b300-b182-11eb-9039-30db6cd9ba55.png)

**Comment by Zrips** [comment, marked as answer]
Small bug, will fix it with next update
